metrotranscom/doorway
# Feat/add partner county limitations

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description
Doorway partners are only allowed to add listings that do not overlap with other current SF Bay Area affordable housing portals. We're limiting the counties that are available to be added via the County drop down box and adding messaging & links to the other portals. 

A subsequent PR will change the seed data to counties represented in the Bay Area jurisdiction (ie not San Mateo county, Alameda county, San Francisco county, and City of San Jose) to ensure that all test data will render correctly in the Partners portal.

## How Can This Be Tested/Reviewed?
Run Partners portal and check that the County field is limited in the Building Details section. Double check the links provided for the other Partner portals work correctly.

![image](https://github.com/metrotranscom/doorway/assets/12400715/cdd56cf0-80a2-4756-9e03-8fc22e131c3c)

![image](https://github.com/metrotranscom/doorway/assets/12400715/1f0e5162-9551-4988-afde-afaf84483e1f)

## Checklist:

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
